### ISSUE TITLE
Custom resolution breaks glowing effect

### ISSUE THREAD

**Issue by Arrocha08** [UltimateBoomer/Resolution-Control#45]
Mod Version: 3.0.0
Minecraft Version: 1.20.1
Fabric API: 0.85.0
Fabric Loader: 0.14.21

Description:
Even in preset resolutions, the glowing effect breaks

https://github.com/UltimateBoomer/Resolution-Control/assets/105840060/62c99d34-9e6f-44db-abaa-4aa663bfc085

**Comment by plushycat** [comment]
Agreed. Happened to me as well and it took me ages to figure out this was the mod behind it.

**Comment by Arrocha08** [comment]
lol, me too

**Comment by ChicknTurtle** [comment]
Hope this can be fixed soon

**Comment by Wartets** [comment]
I have the same problem. On each version of the mod I've used: 1.17 to 1.20. At the moment, I'm playing in 1.20.2, so it would be great to have it set.
